### PR TITLE
Publish the Crowdin monthly January 2021

### PR DESCRIPTION
**Pull request by Julievrz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Publish Crowdin monthly January 2021

### CODE REVIEW

File: news/_posts/translation/do-you-speak-prestashop-january-2021.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+Congratulations! :muscle:
+ 
+![Best translation progress for January 2021](/assets/images/2021/02/build-crowdin-progress-jan21.png)
+
+A huge thank you to all the contributors!
+ 
+Of course, this is highlighting the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active. Indeed, some editing and rewriting could be going on, but the percentage of translation would not be modified (since it is working on strings that are already translated). So let's not forget about the work of other proofreaders! Thanks to you too!
+ 
+## Languages that need (more) proofreaders
```
Comment (LouiseBonnard) [inline news/_posts/translation/do-you-speak-prestashop-january-2021.md]: I'm wondering about this section... it has not changed for a while, should we turn it into a more relevant part while waiting for those projects to gain translations?

File: news/_posts/translation/do-you-speak-prestashop-january-2021.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "Do you speak PrestaShop? – January 2021 edition"
+subtitle: "Where you'll learn that two more languages reached 100%"
```
Comment (LouiseBonnard) [inline news/_posts/translation/do-you-speak-prestashop-january-2021.md]: How about telling which two languages reached 100%? ;-)

**Review by LouiseBonnard — COMMENTED** [review]
Great, thanks @Julievrz! It looks okay to me, just add the publication date in the URL title and edit the one in the header.

File: news/_posts/translation/do-you-speak-prestashop-january-2021.md (hunk 1 above)
Comment (Julievrz) [reply news/_posts/translation/do-you-speak-prestashop-january-2021.md]: I was thinking the same thing. Should we keep it like this for this time and change it next month?

File: news/_posts/translation/do-you-speak-prestashop-january-2021.md (hunk 2 above)
Comment (Julievrz) [inline news/_posts/translation/do-you-speak-prestashop-january-2021.md]:
```suggestion
subtitle: "Where you'll learn that Romanian and Spanish reached 100%"
```

**Pull request merged**